ink/yuki/reward_joi

```diff
@@ -0,0 +1,341 @@
+
+=== yuki_reward_joi ===
+
+TODO: add this scene as an alternative to the original yuki milking scene, if the player chooses the joi option instead.
+
+
+-> select_gender_check ->
+~ setScene(reward_scene, Yuki, "yuki_reward_joi", -> yuki_reward_joi)
+-> scene_setup -> start
+
+= scene_setup
+
+TODO: ask domina about scene setup/ where to find the parameters
+
+->->
+
+= load_gave_reward
+-> scene_setup -> gave_reward
+
+= start
+//this is where the player is asked if they want to give up their reward stone
+
+
+{shuffle:
+    - I just love teaching my puppies how to cum!
+    - It warms my heart when a good puppy wants me to show them how to play with themselves.
+    - Such a curious little {player_name}!
+    
+}
+
+
+{shuffle:
+    - I'll show you how to make yourself feel really good!
+    - Remember, even though i'm going to teach you how to jerkoff, you're still only allowed to cum when a mistress allows it. You wouldn't want to break the rules puppy. 
```

Comment:
I think the reminder not to cum without permission should be stated no matter what, so maybe move it out of the shuffle and put a different line it that is like the first one (emphasis on how much the player will enjoy it)

Reply:
Very true. Good idea!